app/controllers/application_controller.rb

```diff
@@ -1,9 +1,27 @@
-class ApplicationController < ActionController::Base
+
+
+class applicationController < ActionController::Base
```

Comment:
class or module name must be a constant literal
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)